### DISCUSSION TITLE
打开这个插件就看不了评论了吗

### DISCUSSION THREAD

**Discussion by Choi2333** [amll-dev/applemusic-like-lyrics#240] [Q&A]
我把插件更新到最新之后，歌词页面向下就没法滚动了；我记得原来还能滚动到最下面看评论来着？

**Comment by Linho1219** [comment]
AMLL bncm 插件已停止维护，最终版本存档于 <https://github.com/amll-dev/applemusic-like-lyrics-proto>。

现关闭此讨论，敬请谅解。
